File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.py
def solution(s):
    a=0
    b=0
    for i in range(0,len(s)):
        if s[i]=='p' or s[i]=='P':
            a=a+1
        elif s[i]=='y' or s[i]=='Y':
            b=b+1
        else:
            i=i+1
    if a==b:
        return True
    else:
        return False

File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.java
class Solution {
    boolean solution(String s) {
        boolean answer = true;
        int pcnt = 0;
        int ycnt = 0;

        for (int i = 0; i < s.length(); i++) {
            if (s.substring(i, i+1).equalsIgnoreCase("p")) {
                pcnt += 1;
            }
            if (s.substring(i, i+1).equalsIgnoreCase("y")) {
                ycnt += 1;
            }
        }
    
        if (pcnt!=ycnt) {
            answer = false;
        }

        return answer;
    }
}